tensorflow/lite/micro/tools/ci_build/test_esp32.sh

```diff
@@ -26,34 +26,34 @@ pwd
 source tensorflow/lite/micro/tools/ci_build/helper_functions.sh
 
 TARGET=esp
+TARGET_ARCH=xtensa-esp32
 
-# setup esp-idf and toolchains
-echo "Checking out esp-idf..."
-readable_run git clone --recursive --single-branch --branch release/v4.2 https://github.com/espressif/esp-idf.git
-export IDF_PATH="${ROOT_DIR}"/esp-idf
-cd $IDF_PATH
-readable_run ./install.sh
-readable_run . ./export.sh
-cd "${ROOT_DIR}"
+readable_run make -f tensorflow/lite/micro/tools/make/Makefile TARGET=${TARGET} TARGET_ARCH=${TARGET_ARCH} third_party_downloads
 
 # clean all
 readable_run make -f tensorflow/lite/micro/tools/make/Makefile clean
 
+# validate esp32 build for libtensorflow-microlite.a
+readable_run make -j8 -f tensorflow/lite/micro/tools/make/Makefile TARGET=${TARGET} TARGET_ARCH=${TARGET_ARCH} TARGET_TOOLCHAIN_PREFIX=xtensa-esp32-elf-
```

Comment:
For esp32 a no argument invocation of the makefile results in a clean build of the static library.  There are build errors if you choose the 'build' target liike some of the other ci scripts.